OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -1080,17 +1134,14 @@ calcMuscleDynamicsInfo(const SimTK::State& s, MuscleDynamicsInfo& mdi) const
         // Get the quantities that we've already computed.
         const MuscleLengthInfo &mli = getMuscleLengthInfo(s);
         const FiberVelocityInfo &mvi = getFiberVelocityInfo(s);
-        double fiberStateClamped = mvi.userDefinedVelocityExtras[0];
+        double fiberStateClamped =
+            mvi.userDefinedVelocityExtras[MVIIsFiberStateClamped];
```

Comment:
`at`, if that's available - it's hard to keep track of the state of the user variables (too big? too small?)

Reply:
It looks like it is not available. I could switch to standard lib?

Reply:
I'd avoid changing datastructures, but if you still want bounds-checking then an internal `At` helper function is usually how I handle it for OpenSim/SimTK types: 

https://github.com/ComputationalBiomechanicsLab/opensim-creator/blob/0.5.5/src/OpenSimCreator/Utils/OpenSimHelpers.hpp#L104

Reply:
I suggest to do this in a separate PR as well.